cmd/controller-manager/main.go

```diff
@@ -55,13 +55,13 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
+	var metricsAddr, pprofAddr, probeAddr string
 	var enableLeaderElection bool
-	var probeAddr string
 	var itsName string
 	var wdsName string
 	var allowedGroupsString string
-	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The [host]:port from which /metrics is served.")
```

Comment:
Should we move port numbers to some config file ?

Reply:
I see no need to move the _default_ port numbers to a config file, when the _actual_ port numbers are controlled by other files we control that are config-ish (i.e., Helm charts, PostCreateHooks, scripts).